src/lib_ccx/ocr.c

```diff
@@ -1025,6 +1026,9 @@ char *paraof_ocrtext(struct cc_subtitle *sub, const char *crlf, unsigned crlf_le
 	return str;
 }
 #else
+
+struct image_copy;
```

Comment:
What is this for?

Reply:
I got this after running make: `warning: ‘struct image_copy’ declared inside parameter list will not be visible outside of this definition or declaration`